diff --git arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
index fa33c1eff..ea1873ad1 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
+++ b/arch/arm64/boot/dts/amlogic/meson-g12-common.dtsi
@@ -2263,6 +2263,7 @@ watchdog: watchdog@f0d0 {
 				compatible = "amlogic,meson-gxbb-wdt";
 				reg = <0x0 0xf0d0 0x0 0x10>;
 				clocks = <&xtal>;
+				status = "okay";
 			};
 
 			spicc0: spi@13000 {
